### PR TITLE
Update hero run guideline

### PR DESCRIPTION
**Pull request by hdrei**

### CODE REVIEW

File: docs/runjobs/scheduled-jobs/hero-runs.md [hunk 1]
```diff
@@ -41,6 +41,11 @@ through the general contact form, as the "large-scale runs" category.
   something that cannot be done in the normal production queues. As well as
   evidence that you already have it running on LUMI.
 
+## Once the run is granted
+
+Before the actual full machine run, you will be informed about specific settings and small tests to do. The LUMI user support team will be in contact with you about these. 
+Notice that the actual full machine run slots are zero-support sessions, and on the full machine run Sunday your team should be able to manage the run yourselves.
```
Comment (emmanuelumi) [inline docs/runjobs/scheduled-jobs/hero-runs.md]: Please note that support is not available the days full-machine runs are carried out.

File: docs/helpdesk/issues.md [hunk 2]
```diff
@@ -42,6 +42,16 @@ If you suspect that the job has crashed because of a faulty node:
...
 
+- To exclude the faulty node(s) from the resources granted for your job, you can use the Slurm options `-x` or `--exclude`. E.g. to exclude the node nid005038:
+  ```
+  #SBATCH --exclude=nid005038
+  ```
+  Or to exclude the two nodes nid005038 and nid005270:
+  ```
+  #SBATCH --exclude=nid[005038,005270]
+  ```
+
```
Comment (mszpindler) [inline docs/helpdesk/issues.md:53]: I am not in favour of making it general recommendation. It is better to receive updates on faulty nodes in tickets than users handling themselves. Usually exclusion lines preserve in users job scripts for good which hurts scheduling.

Comment (hdrei) [reply docs/helpdesk/issues.md:53]: I see. Suggest to this addition is basically initiated for the needs of FMR's. (They need to be able to figure out themselves how to sort out problems with faulty nodes during their time slot.) 
One option is of course to not add this in docs, but to just inform the people doing FMR's about this separately. 
On the other hand I think this is some basic info that LUMI users should be aware about. I see your point that people might not inform LUST about the faulty nodes, but, this is basic slurm commands and I suppose many people are aware of it already anyway. Maybe we should just emphasize more that please report the problems with nodes, and, also do check at some point if the status of the nodes has changed?

File: docs/runjobs/scheduled-jobs/hero-runs.md (hunk 1 above)
Comment (hdrei) [reply docs/runjobs/scheduled-jobs/hero-runs.md]: This is now changed

**Review by mszpindler — APPROVED** [review]

**Pull request merged**